frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -7,7 +7,65 @@
           <i>{{ currentMap.name }}</i>
         </p>
       </header>
-      <footer v-if="currentMap.type !== 'custom'" class="card-footer">
+      <div v-if="currentMap.reactionList && missingReactionList.length > 0"
+           class="card-content p-4 sidebarCardHover">
+        <div class="content mb-0" @click="showModal = true">
+          Please note that {{ missingReactionList.length }}
+          of the reactions in the model are not shown on the map
+          <span class="icon"><i class="fa fa-info-circle"></i></span>
+        </div>
+        <div v-if="showModal" class="modal is-active">
+          <div class="modal-background" @click="closeModal"></div>
+          <div class="modal-content p-5 column is-6-fullhd is-8-desktop is-10-tablet is-full-mobile
+          has-background-white">
+            <h4 class="title is-size-4 m-0 mb-2"> List of missing and total reactions on the map </h4>
+            <p class="pb-4"> There are {{ missingReactionList.length }}
+              reactions not shown on the map as compared to the model since
+              the map is manually curated and some reactions are not possible
+              to add to the map. The number of reaction shown on the map is
```

Comment:
The part of the text that aims to provide an explanation should be moved to the Documentation page:

```suggestion
              reactions not shown on the map as compared to the model.
              The number of reaction shown on the map is
```

Reply:
1. Should we link to the Documentation?
2. Currently, the documentation says _"While a very high percentage of the reactions in the model are represented on the 2D maps, some may be unavailable."_ Should we elaborate why some may be unavailable, e.g.
_"While a very high percentage of the reactions in the model are represented on the 2D maps, some may be unavailable. 
The maps are manually curated and some reactions are not possible to add."_

Reply:
Yes, and yes:
> While a very high percentage of the reactions in the model are represented on the 2D compartment and subsystem maps, some may be missing. This is partly due to the model being updated much more often than the maps. Also, as the maps are manually curated, in rare occasions some reactions are cannot be added.

Reply:
Didn't we decide to not link to the Documentation page? Below is a screenshot from this issue's description. I thought the text within the parenthesis meant that we should show it in the popup and not link to the Documentation page.

<img width="800" alt="image" src="https://user-images.githubusercontent.com/423498/120215504-1a49d180-c236-11eb-916f-3014509ca985.png">

Reply:
@inghylt and me considered about it but then we thought it might be awkward to have two links, one to the modal table and another to the documentation, almost at the same place.

Reply:
That must be a misunderstanding on my side - I imagined the `icon` to act as the link to the Documentation.
So, the user should be taken to the Documentation page, and to keep things the same as the `Data overlay` and to make it clear that it's not functionality related, the link should be achieved through an `icon`.